packages/registry/src/registry/component-metadata.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
-import {IComponentMetadata, ISimulation, IStyleMetadata} from './types';
+import {IComponentMetadata, ISimulation, IStyleMetadata, ICompInfo} from './types';
 import {isValidSimulationTitle} from '../utils';
 
 export class ComponentMetadata<Props> implements IComponentMetadata<Props> {
   public simulations: Array<ISimulation<Props>> = []; // Initialize with "empty" simulation
   public styles: Map<any, IStyleMetadata> = new Map<any, IStyleMetadata>();
   public reactStrictModeCompliant: boolean = true;
+  public compInfo: ICompInfo = {path: '', exportName: '', baseStylePath: ''};
```

Comment:
Component metadata contains information about the component, now we have information a as subset of information. I would suggest either to remove this nesting and add `exportPath`, `exportName`, and `baseStylePath` directly on the metadata object, or to rename `compInfo` to something like `exportInfo`.